qemu/tests/cfg/snp_multi_vm.cfg

```diff
@@ -0,0 +1,27 @@
+- snp_multi_vm:
+    type = snp_multi_vm
+    kill_vm = yes
```

Comment:
suggest adding start_vm = no here, although we handle this as 'no' when it's not defined as 'yes', but it's easy for anyone to understand we don't want to start vm by VT at the very beginning

Reply:
I am afraid we cannot set start_vm to yes here, because vt will try to create the vm if start_vm = yes automatically. it should be 'no', my previous suggestion was totally based on the more readable code, make it clear that we don't want to start(create) the vm during preprocess. Anyway, it's OK for me too without setting it.